index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      function displayTasks(tasks) {
+        console.log("========================");
+        console.log("現在持っているタスク一覧");
+        console.log("========================");
+        tasks.forEach(function (p, index) {
```

Comment:
forEachのコールバック関数にはアロー関数を使用するのが一般的ですので、覚えておきましょう。
```suggestion
        tasks.forEach( (p, index) => {
```